scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/fgboost/VflGBoostEstimator.scala

```diff
@@ -225,6 +229,6 @@ class VflGBoostEstimator(continuous: Boolean,
         }).toList.asJava)
         .build()
     }.toList
-    flClient.fgbostStub.uploadTreeEval(res.asJava)
+    flClient.fgbostStub.evaluate(res.asJava)
```

Comment:
is this useful?

Reply:
Does `validateLast` update some storage on server? Or just a validation step? I think we can just use the data already on server to calculate the validation result? @qiuxin2012